src/modules/workflows/workflows.js

```diff
@@ -0,0 +1,233 @@
+const express = require('express');
+const { PrismaClient, Prisma } = require('@prisma/client');
+
+const { validateMiddleware } = require('../../middlewares');
+const { CreateProcess } = require('./enums');
+const { createWorkflowDto, updatedWorkflowDto } = require('./dtos');
+
+const router = express.Router();
+
+const prisma = new PrismaClient({
+  // log: ['query', 'info', 'warn', 'error']
+});
+
+// Endpoint to initiate a workflow
+router.post('/', validateMiddleware(createWorkflowDto), async (req, res) => {
+  try {
+    const {
+      body: { styleId, brand, title }
+    } = req;
+
+    const workflow = await prisma.workflow.create({
```

Comment:
Before creating, check if style id is present. 
Else throw error.

Reply:
as discussed, styleIds can be duplicated as per rahul